---
title: "Driver genes affected by subtle mutations, by amplification or homozygous deletion"
output: html_document
---

Ref: [Cancer Genome Landscapes](http://www.sciencemag.org/content/339/6127/1546/suppl/DC1)

**Bert  Vogelstein, Nickolas	Papadopoulos, Victor E.	Velculescu, Shibin	Zhou, Luis A.	Diaz	Jr., Kenneth W.	Kinzler**

Science 29 March 2013: vol. 339 no. 6127 pp. 1546-1558

DOI: 10.1126/science.1235122


Genes were classified as Oncogenes if they had an  Oncogene Score >20% and classified as a Tumor Suppressor Gene (TSG) if the TSG Score was >20% (the 20/20 rule).  The Oncogene  Score was defined as the number of clustered mutations (i.e., missense mutations at the same amino acid or identical in-frame insertions or deletions) divided by the total number of mutations.  The TSG Score was defined as the number of truncating mutations divided by the total number of mutations.   Truncating mutations included nonsense mutations, insertions or deletions that alter the reading frame, splice-site mutations, or mutations at the normal stop codon predicted to result in a longer protein.   When a gene had an oncogene score >20% and a TSG Score >5%, it was classified as a TSG because well-studied oncogenes rarely harbor stop codons.     The major data source for this classification was the [COSMIC database](www.sanger.ac.uk/genetics/CGP/cosmic/).  To be classified as an oncogene, there had to be >10 clustered mutations in this database.  To be classified as a tumor suppressor gene, there had had to be at least 7 inactivating mutations recorded in this database.     In those cases in which  7  to 20 inactivating mutations  were recorded in the COSMIC database, manual curation was performed.  This curation was used to identify other examples of mutations not yet recorded in the COSMIC database and to exclude the most common artifacts encountered in next-generation sequencing, such as mapping errors and high mutation frequencies  observed in normal tissues.   Genes with mutations occurring predominantly in tumors with very high rates of mutation, such as in mismatch-repair deficient tumors or melanomas, were excluded.  As more individual tumors are sequenced in the future, the 20/20 rule can be improved by (i)  considering mutations only in particular tumor types, rather than in all tumor types combined (as done here); (ii) requiring a higher number (e.g., 15) of clustered or inactivating mutations as a threshold for inclusion; and (iii) for genes with thousands of recorded mutations, choose a random subset to calculate the Oncogene Score (if enough tumors are sequenced, all mutations will appear to be clustered)

A gene was classifed as an Oncogene if it was included in the [Cancer Gene Census](www.sanger.ac.uk/genetics/CGP/Census/)  and met the criteria for a high confidence amplfied gene (Class I or II) described in Santarius  et al., Nat Rev Cancer 2010;10(1):59-64.  A gene was classified as a TSG if had at least 10 documented homozygous deletions in the [COSMIC database](http://www.sanger.ac.uk/genetics/CGP/cosmic/) and was not co-deleted with another genes that had at least 10 documented instances of homozygous deletion. The genes in this table include those that are amplified or deleted but are listed as driver genes affected by intragenic alterations or copy number changes.


```{r, echo=FALSE, message=FALSE}
library("plyr");library("dplyr")
library("knitr")
dr <- read.csv("~/Dropbox/Onkobioloogia/Tumorigenees/TableS2A.csv")
amd <- read.csv("~/Dropbox/Onkobioloogia/Tumorigenees/TableS2B.csv")
drivers <- rbind.fill(dr,amd)
kable(drivers, format = "markdown")
```

